src/main/resources/db/migration/V2024091812059__creating_article_and_tag_tables.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE IF NOT EXISTS tag
+(
+    id   SERIAL PRIMARY KEY,
+    name VARCHAR(30) NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS article
+(
+    id                     SERIAL PRIMARY KEY,
+    title                  VARCHAR(70)   NOT NULL,
```

Comment:
ok, I see `NOT NULL` almost everywhere ) in that case I do not understand why do you use nullable in Kotlin